sdk/textanalytics/Azure.AI.TextAnalytics/tests/RecognizeEntitiesTests.cs

```diff
@@ -93,6 +94,37 @@ public async Task RecognizeEntitiesWithSubCategoryTest()
                 if (entity.Text == "last week")
                     Assert.AreEqual("DateRange", entity.SubCategory);
             }
+
+            // Assert the options classes since overloads were added and the original now instantiates a RecognizeEntitiesOptions.
```

Comment:
nice!!
Should we move this to a method that takes `includeStatistics` and optional `modelVersion` so we can call that everywhere?

Reply:
If `options` were a large object or required more advanced drill-downs to assert, I'd agree. But I generally like to keep simple, varied assertions like this in the text case rather than having to navigate all over to find out what the test is doing. But if you really want me to, I can. It'll only save a few lines, though. ;)